src/test/scala/edu/ie3/util/osm/SimpleOsmTestData.scala

```diff
@@ -34,6 +34,13 @@ trait SimpleOsmTestData {
     n2,
     n3
   )
+
+  protected val nodesMap: Map[Long, Node] = Map(
+    n1.id -> n1,
+    n2.id -> n2,
+    n3.id -> n3
+  )
+
```

Comment:
You could also just keep this inside the `"build a polygon correctly"` test case, but I don't have a strong opinion on this

Reply:
I was thinking that it would probably be used more often if we extend the functionality of the utils